sklearn/preprocessing/label.py

```diff
@@ -531,23 +511,15 @@ def label_binarize(y, classes, neg_label=0, pos_label=1,
         data.fill(pos_label)
         Y = sp.csr_matrix((data, indices, indptr),
                           shape=(n_samples, n_classes))
-
     elif y_type == "multilabel-indicator":
         Y = sp.csr_matrix(y)
         if pos_label != 1:
             data = np.empty_like(Y.data)
             data.fill(pos_label)
             Y.data = data
-
-    elif y_type == "multilabel-sequences":
-        Y = MultiLabelBinarizer(classes=classes,
-                                sparse_output=sparse_output).fit_transform(y)
-
-        if sp.issparse(Y):
-            Y.data[:] = pos_label
-        else:
-            Y[Y == 1] = pos_label
-        return Y
+    else:
+        raise ValueError("%s target data is not supported with label "
+                         "binarization" % y_type)
```

Comment:
Is this already handled by [line 638](https://github.com/scikit-learn/scikit-learn/pull/4919/files#diff-1e175ddb0d84aad0a578d34553f6f9c6R637)?